Adjust wording around free / paid / pro / enterprise plans in a few places (#1377)
wandb/docs: content/en/guides/hosting/hosting-options/saas_cloud.md

## Data security

For users on Free or Pro plans, all data is only stored in the shared cloud storage and is processed with shared cloud compute services. Depending on your pricing plan, you may be subject to storage limits.

> Users on an Enterprise plan can [bring their own bucket (BYOB) using the secure storage connector]({{< relref "/guides/hosting/data-security/secure-storage-connector.md" >}}) at the [team level]({{< relref "/guides/hosting/data-security/secure-storage-connector.md#configuration-options" >}}) to store their files such as models, datasets, and more. You can configure a single bucket for multiple teams or you can use separate buckets for different W&B Teams. If you do not configure secure storage connector for a team, that data is stored in the shared cloud storage.

Author: ```suggestion
Users on an Enterprise plan can [bring their own bucket (BYOB) using the secure storage connector]({{< relref "/guides/hosting/data-security/secure-storage-connector.md" >}}) at the [team level]({{< relref "/guides/hosting/data-security/secure-storage-connector.md#configuration-options" >}}) to store their files such as models, datasets, and more. You can configure a single bucket for multiple teams or you can use separate buckets for different W&B Teams. If you do not configure BYOB for a team, the team's data is stored in the shared cloud storage.
```

You are responsible for ensuring that your deployment complies with your organization's policies and [Security Technical Implementation Guidelines (STIG)](https://en.wikipedia.org/wiki/Security_Technical_Implementation_Guide), if applicable.

## Identity and access management (IAM)

> If you are on an Enterprise plan, you can use identity and access managements capabilities for secure authentication and effective authorization in your W&B Organization. The following features are available for IAM in Multi-tenant Cloud:

Author: ```suggestion
If you are on an Enterprise plan, enhanced identity and access managements capabilities allow for secure authentication and effective authorization for your W&B deployment:
```

> If you are on an Enterprise plan, you can use the identity and access managements capabilities for secure authentication and effective authorization in your W&B Organization. The following features are available for IAM in Multi-tenant Cloud:

Author: ```suggestion
If you are on an Enterprise plan, you can use identity and access managements capabilities for secure authentication and effective authorization in your W&B Organization. The following features are available for IAM in Multi-tenant Cloud:
```

Security controls for Multi-tenant Cloud are periodically audited internally and externally. Refer to the [W&B Security Portal](https://security.wandb.ai/) to request the SOC2 report and other security and compliance documents.

## Next steps

> Access [Multi-tenant Cloud directly](https://wandb.ai) to get started with Free or Pro. To start with the Enterprise plan, submit [this form](https://wandb.ai/site/for-enterprise/multi-tenant-saas-trial).

Author: ```suggestion
Access [Multi-tenant Cloud directly](https://wandb.ai) to get started with most features for free. To try out enhanced data security and IAM features, [request an Enterprise trial](https://wandb.ai/site/for-enterprise/multi-tenant-saas-trial).
```
